apps/style/craft/style.scss

```diff
@@ -163,6 +163,18 @@ $text-shadow-gray: #5a5a5a;
   @include arrow-icon("#{$root}Sliced_Parts/MC_Up_Arrow_Icon.png");
   background-size: 28px 20px !important;
 }
+.scroll-up-btn {
+  @include arrow-icon("#{$root}CDO_MC_ScrollUp.png");
```

Comment:
![image](https://cloud.githubusercontent.com/assets/206973/20345869/095856d6-abf1-11e6-9435-93ec1a507f63.png)

(Cool how it works though! Gotta love smaller asset sizes)